force_wfmanager/left_side_pane/tests/test_kpi_specification_model_view.py

```diff
@@ -44,3 +44,12 @@ def test_update_combobox_values(self):
 
     def test_label(self):
         self.assertEqual(self.kpi_specification_mv.label, "KPI")
+
+    def test_name_change(self):
+        self.data_source1.output_slot_info = [OutputSlotInfo(name='T1')]
+        self.kpi_specification_mv.model.name = ''
+        self.assertEqual(self.kpi_specification_mv.label,
+                         'KPI')
+        self.kpi_specification_mv.model.name = 'T1'
+        self.assertEqual(self.kpi_specification_mv.label,
+                         'KPI: '+self.kpi_specification_mv.model.name)
```

Comment:
Don't compute the answer if you know precisely what it will be (I think "KPI: T1" in this case.)